Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/IDE/Wing/readme.md

```diff
@@ -1,32 +1,38 @@
 # O3DE DCCsi, Wing Pro IDE
 
+[Wing Pro](https://wingware.com/) is a Python IDE that can be used to develop, test, and debug Python code written for [Open 3D Engine (O3DE)](https://www.o3de.org/), an open source realtime3D engine, for video games and other uses cases.
```

Comment:
Is there a reason to provide specific IDE instructions? 

I would also argue O3DE uses PyCharm and other IDEs more than Wing IDE and we should lean towards IDEs that support our major platforms which include Linux (and to some extent PyCharm).

Reply:
Yes, each IDE has sometimes very different configuration setups and/or capabilities.  Some of this targets a TA that may be a novice Python user, and this readme is in line with the types of similar guide you might find as a TA looking for this information for setting up Wing for Maya, etc.

I know that when I got started (with Python and IDEs), it would have been great to has access to a guide like this.

We'd have a readme for PyCharm, it has its own steps to be configured to be used well with O3DE - because by default it expects a system interpretter, not one imbedded within a code base like this. 

My intent has been to target Wing, PyCharm and VScode setup.  We can flesh out PyCharm next, but I'd rather first move the DCCsi to a o3de-tachart repo, and then pull Wing (and the other tools) into sub-gems before going further.

Reply:
@lmbr-pip If I was starting the DCCsi from scratch today, I would have done PyCharm first.  But I am a Wing user, this is a prototype, and Ben has been more focused on PyCharm but that work is not in the current branch.  I agree with you thought that most people probably do use PyCharm, and that as we move forward we need to widen platform support (although, pretty sure that wing runs on mac and linux. and also that there is a free community version we could also support hopefully.)